docs/modules/migrate/pages/lts.adoc

```diff
@@ -0,0 +1,121 @@
+= Long-term Support Releases
```

Comment:
not strong on this but we can do this
```suggestion
= Long-term Support (LTS) Releases
```

for this to appear in navigation

Reply:
It is best to avoid acronyms in headings where possible - if you can provide a reason to include it here, I won't argue; however, I would prefer not to, the acronym is given in the description and I believe most people would be aware of it